lib/matplotlib/style/core.py

```diff
@@ -130,6 +145,20 @@ def use(style):
                              if k not in STYLE_BLACKLIST}
             elif style in library:
                 style = library[style]
+            elif "." in style:
+                pkg, name = style.rsplit(".")
+                try:
+                    path = (importlib_resources.files(pkg)
+                            / f"{name}.{STYLE_EXTENSION}")
+                    style = _rc_params_in_file(path)
+                except (ModuleNotFoundError, IOError) as exc:
+                    # There is an ambiguity whether a dotted name refers to a
```

Comment:
I guess it would be good to test this block too unless you think the behaviour isn't decided?

Reply:
Sure, writing a test now.